examples/optuna_search_cv_simple.py

```diff
@@ -0,0 +1,40 @@
+"""
+Optuna example that optimizes a classifier configuration using OptunaSearchCV.
+
+In this example, we optimize a classifier configuration for Iris dataset using OptunaSearchCV.
+Classifier is from scikit-learn.
+
+We have the following two ways to execute this example:
+
+(1) Execute this code directly.
+    $ python optuna_search_cv_simple.py
+
+
+(2) Execute through CLI.
+    $ STUDY_NAME=`optuna create-study --direction maximize --storage sqlite:///example.db`
+    $ optuna study optimize optuna_search_cv_simple.py objective --n-trials=100 --study \
+      $STUDY_NAME --storage sqlite:///example.db
```

Comment:
The second way does not work correctly because `optuna_search_cv_simple.py` does not have `objective` function. I don't think `OptunaSearchCV` can be executed from CLI, so please remove it.